src/App.vue

```diff
@@ -73,17 +78,9 @@ export default {
       featureInfo: true,
       searchable: true,
       pathControls: true,
+      layerControl: true,
       minZoom: 4,
-      availableSpecies : {
-        "Human Female":{taxo: "NCBITaxon:9606", biologicalSex: "PATO:0000383", iconClass:"mapicon-icon_human", displayWarning:true},
-        "Human Male":{taxo: "NCBITaxon:9606", biologicalSex: "PATO:0000384", iconClass:"mapicon-icon_human", displayWarning:true},
-        "Rat":{taxo: "NCBITaxon:10114", iconClass:"mapicon-icon_rat", displayWarning:false, displayLatestChanges:true},
-        "Mouse":{taxo: "NCBITaxon:10090", iconClass:"mapicon-icon_mouse", displayWarning:true},
-        "Kember":{taxo: "ABI:1000001", displayWarning:true},
-        "Pig":{taxo: "NCBITaxon:9823", iconClass:"mapicon-icon_pig", displayWarning:true}, 
-        "Cat":{taxo: "NCBITaxon:9685", iconClass:"mapicon-icon_cat", displayWarning:true},
-        "Functional Connectivity": {id: "fc-cardio", displayWarning:true},
-        "digestive tract":{taxo: "digestive tract", displayWarning:true}
+      availableSpecies : {"Functional Connectivity":{taxo: "FunctionalConnectivity", displayWarning:true}
```

Comment:
Keep the previous list and replace the functional connectivity with the new one please.